base_classes/nyaml/NXbeam.yaml

```diff
@@ -172,52 +172,17 @@ NXbeam(NXobject):
   incident_polarization(NX_NUMBER):
     unit: NX_ANY
     doc: |
-      Incident polarization as a Stokes vector
-      on entering beamline component
+      Polarization vector on entering beamline component
     dimensions:
       rank: 2
       dim: [[1, nP], [2, 2]]
-    \@units:
-      type: NX_CHAR
-      doc: |
-        The units for this observable are not included in the NIAC list.
-        Responsibility on correct formatting and parsing is handed to the user
-        by using `NX_ANY`. Correct parsing can still be implemented by using
-        this attribute.
-        
-        | Fill with:
-        
-        * The unit unidata symbol if the unit has one (Example: T for the unit of magnetic flux density tesla).
-        * The unit unidata name if the unit has a name (Example: farad for capacitance).
-        * A string describing the units according to unidata unit operation notation, if the unit is a complex combination of named units and
-          does not have a name.
-        
-        Example: for lightsource brilliance (SI) 1/(s.mm2.mrad2).
-        Here: SI units are V2/m2.
```

Comment:
Maybe I missed the discussion about this, but shouldn't we keep this for clarity and the possibility to report composite units?

Reply:
Good point, I wasn't sure if you also wanted to get rid of them. I readded them.